src/Config.php

```diff
@@ -0,0 +1,343 @@
+<?php namespace Rollbar;
+
+use Rollbar\Payload\Payload;
+use Rollbar\Payload\Level;
+
+if (!defined('ROLLBAR_INCLUDED_ERRNO_BITMASK')) {
+    define(
+        'ROLLBAR_INCLUDED_ERRNO_BITMASK',
+        E_ERROR | E_WARNING | E_PARSE | E_CORE_ERROR | E_USER_ERROR | E_RECOVERABLE_ERROR
+    );
+}
+
+class Config
+{
+    private $accessToken;
+    /**
+     * @var DataBuilder
+     */
+    private $dataBuilder;
+    private $configArray;
+    /**
+     * @var TransformerInterface
+     */
+    private $transformer;
+    /**
+     * @var FilterInterface
+     */
+    private $filter;
+    private $minimumLevel;
+    /**
+     * @var ResponseHandlerInterface
+     */
+    private $responseHandler;
+    /**
+     * @var \Rollbar\Senders\SenderInterface
+     */
+    private $sender;
+    private $reportSuppressed;
+    /**
+     * @var callable
+     */
+    private $checkIgnore;
+    private $error_sample_rates = array();
+    private $mt_randmax;
+
+    private $included_errno = ROLLBAR_INCLUDED_ERRNO_BITMASK;
+
+    public function __construct(array $configArray)
+    {
+        $this->updateConfig($configArray);
+
+        if (isset($configArray['error_sample_rates'])) {
+            $this->error_sample_rates = $configArray['error_sample_rates'];
+        }
+
+        $levels = array(E_WARNING, E_NOTICE, E_USER_ERROR, E_USER_WARNING,
+            E_USER_NOTICE, E_STRICT, E_RECOVERABLE_ERROR);
+        // PHP 5.3.0
+        if (defined('E_DEPRECATED')) {
+            $levels = array_merge($levels, array(E_DEPRECATED, E_USER_DEPRECATED));
+        }
+        $curr = 1;
```

Comment:
It's used on line 65 to set the error levels that don't exist to 1, so they always get sent.